docs/about/CHANGELOG.md

```diff
@@ -0,0 +1,278 @@
+---
```

Comment:
Normally we would include this in the release notes and not commit it. Github actions also automatically generates the list of changes since the last release for us.

Reply:
ok - hadnt seen it in the release notes so far - happy to take it out